content/_index.md

```diff
@@ -8,3 +8,6 @@ This is the landing page for Platen, a project for providing scaffolding of digi
 tools and other interactive web books. Platen shortens the distance between creators and digital
 versions of their texts that are accessible, interactive, responsive, and---above all---delightful
 to author and maintain.
+<br />
+![button:Get Started with Platen](https://platen.io/getting-started/happy-path-setup/)
```

Comment:
Let's use the site relative url, so it resolves correctly in local dev and on PR previews.

```suggestion
![button:Get Started with Platen](/getting-started/happy-path-setup)
```